R/run_example.R

```diff
@@ -121,13 +134,32 @@ run_example <- function(){
 
         #####################Create third panel################################
         shiny::tabPanel(
-          "Feedback Types",
+          "Tables and tabs",
           value = "panel3",
           gov_layout(
             size = "two-thirds",
-            backlink_Input("back2"),
+            backlink_Input("back1"),
             heading_text("Page 3", size = "l"),
-            label_hint("label3", "These are some examples of the types of user
+            label_hint("label3", "These are some examples of using tabs and
+                       tables"),
+            heading_text("govTable", size = "s"),
+            shinyGovstyle::govTable(
+              "tab1", example_data, "Test", "l", num_col = c(2,3),
+              width_overwrite = c("one-half", "one-quarter", "one-quarter")),
+            heading_text("govTabs", size = "s"),
+            shinyGovstyle::govTabs("tabsID", data, "tabs")
+          )
+        ),
+
+        #####################Create feedback panel################################
+        shiny::tabPanel(
+          "Feedback Types",
+          value = "panel-feedback",
+          gov_layout(
+            size = "two-thirds",
+            backlink_Input("back2"),
```

Comment:
Should this be back3? Seems odd it jumps to tab2 from tab4

Reply:
Should also be updated. And added a button on panel3 to go to the fourth tab as well.